compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/node/RootNodeOwner.skiko.kt

```diff
@@ -268,15 +269,16 @@ internal class RootNodeOwner(
 
     /**
      * Perform hit test and return the [InteropView] associated with the resulting
-     * [PointerInputModifierNode] node in case it is a [InteropViewAnchorModifierNode], otherwise null.
+     * [PointerInputModifierNode] node in case it is a [Modifier.pointerInteropFilter],
+     * otherwise null.
      */
     fun hitTestInteropView(position: Offset): InteropView? {
         val result = HitTestResult()
         owner.root.hitTest(position, result, true)
 
-        val node = result.lastOrNull() as? InteropViewAnchorModifierNode ?: return null
-
-        return node.interopView
+        val last = result.lastOrNull() as? BackwardsCompatNode
+        val node = last?.element as? InteropPointerInputModifier
+        return node?.interopView
```

Comment:
Is there a specific reason we need to use BackwardsCompatNode instead of new ModifierNode/ModifierElement approach?

Reply:
`git blame` says, that it's 2020-year code from cae70892d88aea7aa222d5b46a407d1cdfe9ab73 before modifier refactor. I'm strongly inclined to redo it in a modern approach.

Reply:
Will do that after upstreaming/commonization